Utilities/Mergers/include/Mergers/MergerConfig.h

```diff
@@ -16,71 +16,49 @@
+namespace o2::mergers
 {
 
 // This is a set of Mergers' options that user can choose from.
 
-enum class OwnershipMode { // todo better name
-  Full,                    // Mergers expect full objects each time.
-  Integral                 // Mergers expect objects' differences (what has changed since the previous was sent).
-};
-
-enum class MergingMode {
-  Binwise,     // Bins of histograms are added, TTree branches are attached, objects inside TCollections are merged correspondingly.
-  Timewise,    // NOT SUPPORTED YET. Arriving objects are merged into one T*****, ordered in time.
-  Concatenate, // Arriving objects are merged into one TObjArray, no particular order.
+enum class InputObjectsTimespan {
+  FullHistory,   // Mergers expect objects with all data accumulated so far each time.
+  LastDifference // Mergers expect objects' differences (what has changed since the previous were sent).
 };
 
-enum class MergingTime {
-  AfterArrival,      // Merging just after new object arrives.
-  WhenXInputsCached, // Merging when 0 < X < 1 of the inputs are cached.
-                     //  EachNSeconds, // NOT SUPPORTED. Merging the most up-to-date inputs each N seconds //todo: maybe later (dpl makes it hard to have two timers)
-  BeforePublication  // Merging just before publication.
-};
-
-enum class Timespan {
-  FullHistory,    // Merged object should consist of all the partial data that Mergers received..
-  LastDifference, // Merged object should consist of only data received after last publication. Merged object is reset after published.
-  //  MovingWindow // NOT SUPPORTED YET. Merged object should consist of object from last N seconds. Inherit MergeInterface and implement time().
+enum class MergedObjectTimespan {
+  // Merged object should be an sum of differences received since the beginning
+  // or a sum of latest versions of objects received on each input.
+  FullHistory,
+  // Merged object should be an sum of differences received after last publication.
+  // Merged object is reset after published. It won't produce meaningful results
+  // when InputObjectsTimespan::FullHstory is set.
+  LastDifference
 };
 
 enum class PublicationDecision {
   EachNSeconds,       // Merged object is published each N seconds.
-  WhenXInputsUpdated, // Merged object is published when 0 < X < 1 of the inputs are updated since previous publication.
-  // AfterEachMerge // NOT SUPPORTED. todo
 };
 
 enum class TopologySize {
   NumberOfLayers, // User specifies the number of layers in topology.
   ReductionFactor // User specifies how many sources should be handled by one merger (by maximum).
 };
 
-enum class UnpackingMethod {
-  NoUnpackingNeeded, // Merger treats object as it is.
-  TCollection,       // NOT SUPPORTED YET. Merger treats each object as TCollection and merges each member accordingly. todo
-};
-
 template <typename V, typename P = double>
 struct ConfigEntry {
   V value;
-  P param;
+  P param = double();
```

Comment:
```suggestion
  P param = P();
```
`P` is not necessarily `double`, only the default